circle.yml

```diff
@@ -1,16 +1,18 @@
 machine:
-  timezone: America/Los_Angeles
   php:
-    version: 5.5.0
+    version:
+      5.5.31
 
 dependencies:
   override:
     - mkdir -p build/logs
+    - echo "date.timezone = America/Los_Angeles" > /opt/circleci/php/$(phpenv global)/etc/conf.d/date.ini
     - composer update --prefer-stable --prefer-lowest --no-interaction
-    - sed -i 's/^;//' ~/.phpenv/versions/$(phpenv global)/etc/conf.d/xdebug.ini
```

Comment:
Xdebug is installed for all versions of PHP at CircleCI.

https://circleci.com/docs/1.0/language-php/#xdebug